mobile/lib/routing/router.dart

```diff
@@ -106,7 +106,11 @@ part 'router.gr.dart';
       guards: [AuthGuard, DuplicateGuard],
       transitionsBuilder: TransitionsBuilders.slideBottom,
     ),
-    AutoRoute(page: AlbumViewerPage, guards: [AuthGuard, DuplicateGuard]),
+    AutoRoute(
+      page: AlbumViewerPage,
+      guards: [AuthGuard, DuplicateGuard],
+      initial: true,
```

Comment:
What affect does this have?

Reply:
Leftover changes while trying other solutions :-)
Reverted these changes